org-ql.el

```diff
@@ -519,6 +519,13 @@ Or, when possible, fix the problem."
 		  (org-ql--sanity-check-form (cdr elem)))
 	     else do (check elem))))
 
+(defun org-ql--inner-query-predicate (preprocessedquery)
```

Comment:
What's the purpose of this function?  Preambles won't be used for these ancestor/parent "sub-queries," so why is `--query-preamble` being called?

Reply:
My mistake. It looked like `--query-preamble` might have made some changes to the query in some way that might be needed. Removed this function.